internal/kconfig/kconfig.go

```diff
@@ -78,7 +78,7 @@ func Parse(source io.ReaderAt) (map[string]string, error) {
 // Golang translation of libbpf bpf_object__process_kconfig_line():
 // https://github.com/libbpf/libbpf/blob/fbd60dbff51c870f5e80a17c4f2fd639eb80af90/src/libbpf.c#L1874
 // It does the same checks but does not put the data inside the BPF map.
-func processKconfigLine(line string, m map[string]string) error {
+func processKconfigLine(line string, m map[string]string, filteredConfigs map[string]any) error {
```

Comment:
This function should take `line []byte` and use `bytes.HasPrefix`, `bytes.Cut` and so on. This avoids allocating `string`s for lines we're not interested in.